pandas/core/groupby/groupby.py

```diff
@@ -2323,8 +2322,20 @@ def first(self, numeric_only: bool = False, min_count: int = -1):
 
         Examples
         --------
-        >>> df = pd.DataFrame(dict(A=[1, 1, 3], B=[None, 5, 6], C=[1, 2, 3]))
+        >>> df = pd.DataFrame(dict(A=[1, 1, 3], B=[None, 5, 6], C=[1,2,3],
```

Comment:
Nit: looks like he spacing for the list assigned to `C` isn't the same as the original